### PR TITLE
chore: Fix lint warnings

### PR DESCRIPTION
**Pull request by nauaros**
## Summary

Resolve all current lint warnings.

### CODE REVIEW

**Review by erenbesel — APPROVED** [review]

File: AdyenCard/Form/Installments/FormCardInstallmentsItem.swift [hunk 1]
```diff
@@ -31,7 +31,8 @@ internal final class FormCardInstallmentsItem: BaseFormPickerItem<InstallmentEle
     }
     
     /// Default picker option.
-    private lazy var oneTimePaymentElement: InstallmentElement = .init(kind: .plan(.oneTime), localizationParameters: localizationParameters)
+    private lazy var oneTimePaymentElement: InstallmentElement =
+        .init(kind: .plan(.oneTime), localizationParameters: localizationParameters)
```
Comment (descorp) [inline AdyenCard/Form/Installments/FormCardInstallmentsItem.swift:35]: Why not `lazy var oneTimePaymentElement = InstallmentElement(...)` ?

File: AdyenDropIn/Views/ModalToolbar.swift [hunk 2]
```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2021 Adyen N.V.
+// Copyright (c) 2022 Adyen N.V.
```
Comment (descorp) [inline AdyenDropIn/Views/ModalToolbar.swift:2]: this thing again 😅

File: AdyenDropIn/Views/ModalToolbar.swift [hunk 3]
```diff
@@ -115,9 +115,11 @@ internal final class ModalToolbar: UIView, AnyNavigationBar {
         ])
     }
 
-    private lazy var safeAreaLeadingAnchor: NSLayoutXAxisAnchor = style.toolbarMode == .natural ? safeAreaLayoutGuide.leadingAnchor : safeAreaLayoutGuide.leftAnchor
+    private lazy var safeAreaLeadingAnchor: NSLayoutXAxisAnchor =
```
Comment (descorp) [inline AdyenDropIn/Views/ModalToolbar.swift:118]: why we need `: NSLayoutXAxisAnchor` ?

File: Adyen/UI/List/ListFooterView.swift [hunk 4]
```diff
@@ -47,7 +47,10 @@ internal final class ListFooterView: UIView {
         return stackView
     }()
 
-    private lazy var titleContainerView: UIView = titleBackgroundView.adyen.wrapped(with: UIEdgeInsets(top: 12, left: 16, bottom: -12, right: -16))
+    private lazy var titleContainerView: UIView = titleBackgroundView.adyen.wrapped(with: UIEdgeInsets(top: 12,
+                                                                                                       left: 16,
+                                                                                                       bottom: -12,
+                                                                                                       right: -16))
```
[the remark this answers is not in the document]
Comment (mohammedDehairy) [reply Adyen/UI/List/ListFooterView.swift:53]: you can disable the swift formatter rule that overwrites this in the `.swiftformat` file:

```
--disable blankLinesAtStartOfScope,blankLinesAtEndOfScope,unusedArguments,redundantSelf,wrapMultilineStatementBraces,extensionAccessControl,redundantClosure
--swiftversion "5.2"
--commas inline
--ranges nospace
--trimwhitespace nonblank-lines
--decimalgrouping none
```

to disable this rule https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#redundantclosure

[the remark this answers is not in the document]
Comment (nauaros) [reply Adyen/UI/List/ListFooterView.swift:53]: Also `Internal/format.sh` right?

**Review by erenbesel — APPROVED** [review]

**Review by descorp — APPROVED** [review]

**Pull request merged**